source/index.slim

```diff
@@ -20,7 +20,13 @@ nav_logo_class: 'is-animated'
 
 - content_for :javascripts
   javascript:
-    $('#HomepageHero').load(function() {
-      var svg = $('#HomepageHero').contents().find('svg')[0];
-      window.homepage.animateLines(svg);
-    })
+    var heroWrapper = $('#HomepageHero');
+    var objectElement = $('#HomepageHeroSVG');
+
+    objectElement
+      .on('load', function() {
+        var svg = objectElement.contents().find('svg')[0];
+
+        window.homepage.parallax(svg, heroWrapper[0]);
+      })
+      .attr('data', objectElement.attr('data'));
```

Comment:
There's no guarantee of that.
This exists as an hack that should have been explained in the commit message.
I'll go with a different approach and update the commit message properly